pest/src/lib.rs

```diff
@@ -58,8 +58,14 @@
 
 extern crate memchr;
 
+pub use error::Error;
```

Comment:
I would generally publish these after `mod` definitions. Was this generated by `rustfmt`?

Reply:
This was formatted with `rustfmt`. I can put it back if you want.